src/js/tabs/setup.js

```diff
@@ -265,7 +269,25 @@ setup.initialize = function (callback) {
             }
         };
 
+        const showFirmwareInfo = function() {
+            MSP.send_message(MSPCodes.MSP_STATUS_EX, false, false, function() {
+                // Firmware info
+                msp_api_e.text([FC.CONFIG.apiVersion]);
+                build_info_e.text([FC.CONFIG.buildInfo]);
+
+                if(FC.CONFIG.buildOptions.length > 0) {
+                    build_opt_e.text = "";
+                    for (let i = 0; i < FC.CONFIG.buildOptions.length; i++) {
+                        build_opt_e.append(FC.CONFIG.buildOptions[i], ' ');
```

Comment:
Removing one sonar cloud issues (we can ignore the bugs in these case)
```suggestion
                    for (const buildOption of FC.CONFIG.buildOptions) {
                        build_opt_e.append(buildOption, ' ');
```

Reply:
great change, done